app/src/main/res/values/strings.xml

```diff
@@ -490,6 +490,8 @@
     <string name="no_saving_account">Saving Accounts</string>
     <string name="no_loan_account">Loan Accounts</string>
     <string name="no_sharing_account">Sharing Accounts</string>
+    <string name="authentication_required">Sign in using fingerprint</string>
+    <string name="text_description_biometrics_dialog">Scan your fingerprint</string>
```

Comment:
What about renaming these strings to: `sing_in_fingerprint` and `scan_your_fingerprint`?